packages/flutter/test/material/slider_test.dart

```diff
@@ -3507,6 +3507,78 @@ void main() {
     );
   }, variant: TargetPlatformVariant.desktop());
 
+  testWidgets('Value indicator disappears after a bit when the slider is clicked, not dragged, on desktop', (WidgetTester tester) async {
+    double currentValue = 0.5;
+    final FocusNode focusNode = FocusNode();
+
+    Widget buildApp({bool enabled = true}) {
+      return MaterialApp(
+        theme: ThemeData(
+          sliderTheme: const SliderThemeData(
+            showValueIndicator: ShowValueIndicator.always,
+          ),
+        ),
+        home: Material(
+          child: Center(
+            child: StatefulBuilder(builder: (BuildContext context, StateSetter setState) {
+              return Slider(
+                value: currentValue,
+                focusNode: focusNode,
+                divisions: 5,
+                label: currentValue.toStringAsFixed(1),
+                onChanged: enabled
+                    ? (double newValue) {
+                  setState(() {
+                    currentValue = newValue;
+                  });
+                }
+                    : null,
+              );
+            }),
+          ),
+        ),
+      );
+    }
+
+    await tester.pumpWidget(buildApp());
+
+    // Slider does not show value indicator before being clicked
+    await tester.pumpAndSettle();
+    expect(focusNode.hasFocus, false);
+    RenderBox valueIndicatorBox = tester.renderObject(find.byType(Overlay));
+    expect(
+      valueIndicatorBox,
+      isNot(paints..path(color: const Color(0xff000000))..paragraph()),
```

Comment:
Just wondering what the code here does : `isNot(paints..path(color: const Color(0xff000000))..paragraph())`.
Specifically the  _..paragraph_  which looks like a new addition. Is this the change you needed to add in order to detect Overlay visibility?